app/views/dvr/dvrFiles.scala.html

```diff
@@ -34,10 +34,11 @@
   govukSummaryList: GovukSummaryList,
   govukRadios: GovukRadios,
   govukErrorSummary: GovukErrorSummary,
-  formWithCSRF: FormWithCSRF
+  formWithCSRF: FormWithCSRF,
+  govukInsetText: GovukInsetText
 )
 
-@(model: AvailableRequestDetailedValuation, startCheckForm: Form[StartCheckForm])(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
+@(model: AvailableRequestDetailedValuation, startCheckForm: Form[StartCheckForm], rateableValue: Option[Long])(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
```

Comment:
can you add `rateableValue` to the `model` instead please.

Reply:
yes, please, well spotted Habeeb

Reply:
Yeah I just had a call with habeeb as my commit didn't answer this at all - it seems my brain really still hasn't switched on 😄